src/NTS/NtsServiceCollectionExtensions.cs

```diff
@@ -20,4 +23,22 @@ public static IServiceCollection ConfigureNtsBlazor(this IServiceCollection serv
 
         return services;
     }
+
+    public static IServiceCollection ConfigureAuthentication(
+        this IServiceCollection services,
+        IConfiguration configuration
+    )
+    {
+        services
+            .AddAuthentication(options =>
+            {
+                options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = GoogleDefaults.AuthenticationScheme;
+            })
+            .AddCookie()
```

Comment:
This should be encapsulated in the `Not.Authentication` and the authentication builder instance should be provided in the same way as the `options => { ... }` argument above. This approach encapsulates stuff that the caller doesn't need to know such as `AddCookie` and `AddAuthorization`, but still allows the caller to configure different authentication providers.